lib/newspaper_works/ingest/ndnp/batch_ingester.rb

```diff
@@ -38,8 +39,17 @@ def self.batch_path(options, cmd_name)
           path
         end
 
+        def self.xml_path(path)
+          return path unless File.directory?(path)
+          batch_xml_path = Find.find(path).select do |f|
+            f.downcase.end_with?('batch_1.xml')
+          end
+          batch_xml_path[0]
+        end
+
         def initialize(path)
-          @path = xml_path(path)
+          @path = self.class.xml_path(path)
+          raise IOError, 'No batch file found: #{path}' if @path.empty?
```

Comment:
I think we need to use double-quotes here for `#{path}` to be interpolated correctly?

Reply:
Yes, fixed in f40bf67762537a0abd84dbeadbb11144cf9972d6